### PR TITLE
Fix legend display

### PR DESCRIPTION
**Pull request by JulienMattiussi**
- Limit legend to same scope fields
- Remove legends from graphic list
- Fix graphLink 'voir les détails' display for home and for legends

### CODE REVIEW

File: src/common/scope.js [hunk 1]
```diff
@@ -2,6 +2,20 @@ export const SCOPE_DATASET = 'dataset';
 export const SCOPE_GRAPHIC = 'graphic';
 export const SCOPE_COLLECTION = 'collection';
 export const SCOPE_DOCUMENT = 'document';
+
+export const sameScope = (scopeA, scopeB) => {
```
Comment (jdemangeon) [inline src/common/scope.js]: This function return true in every case no :grin: ?

If a & b  = SCOPE_DOCUMENT
if a & b = SCOPE_COLLECTION
if a = SCOPE_COLLECTION & b = SCOPE_DOCUMENT 
if b = SCOPE_COLLECTION & a = SCOPE_DOCUMENT

**Review by jdemangeon — CHANGES_REQUESTED** [review]
I don't understand all you've developed, but I trust you!

File: src/common/scope.js (hunk 1 above)
Comment (JulienMattiussi) [reply src/common/scope.js]: For these cases, yes
but when a = SCOPE_DATASET and b = SCOPE_GRAPHIC, then it's false

**Comment by JulienMattiussi** [comment]
> I don't understand all you've developed, but I trust you!

Basically, before, Lodex was checking that the field was a graphic to set somes specific displays.
But after, I replaced it by a check on the Scope : The field is in the graphic page
It's wrong because all the fields in the graphic page are not graphics
So now, I changed the tests again to fit with reality (legends are not graphics field, and some cloned field can be here and should be considered as graphics)

File: src/common/scope.js (hunk 1 above)
Comment (jdemangeon) [reply src/common/scope.js]: OK, so can you use includes instead, and rename the function because it not only test for equality

Comment (JulienMattiussi) [reply src/common/scope.js]: Use fantastic functional syntax from JulienD's big brain

**Review by jdemangeon — APPROVED** [review]

**Pull request merged**